### PR TITLE
chore: removing input-descriptor-to-credential application

### PR DESCRIPTION
**Pull request by artursudnik**

### CODE REVIEW

File: common/config/rush/pnpm-lock.yaml
Comment (jrhender) [inline common/config/rush/pnpm-lock.yaml:1]: Do we want to delete the `pnpm-lock.yaml` file?

**Review by jrhender — COMMENTED** [review]
Looks good, one comment about pnpm-lock file

File: common/config/rush/pnpm-lock.yaml
Comment (artursudnik) [reply common/config/rush/pnpm-lock.yaml:1]: It is not deleted. GitHub just hides it in review as it is treated as a non-human-readable file. You can still see the changes after clicking `Load diff`

Comment (jrhender) [reply common/config/rush/pnpm-lock.yaml:1]: Ah thanks, my mistake.

**Review by jrhender — APPROVED** [review]
Looks good, thanks Artur

**Pull request merged**